pkg/controller/jivavolume/jivavolume_controller.go

```diff
@@ -0,0 +1,709 @@
+		"openebs.io/persistent-volume": pv,
+	}
+}
+
+// TODO: Add code to configure resource limits, nodeAffinity etc.
+func createReplicaStatefulSet(r *ReconcileJivaVolume, cr *jv.JivaVolume,
+	reqLog logr.Logger) error {
+
+	var (
+		err                            error
+		replicaCount                   int32
+		stsObj                         *appsv1.StatefulSet
+		blockOwnerDeletion, controller = true, true
+	)
+	rc, err := strconv.ParseInt(cr.Spec.ReplicationFactor, 10, 32)
+	if err != nil {
+		return fmt.Errorf("failed to create replica statefulsets: error while parsing RF: %v, err: %v",
+			cr.Spec.ReplicationFactor, err)
+	}
+
+	replicaCount = int32(rc)
+	prev := true
+
+	size := strings.Split(cr.Spec.Capacity, "i")[0]
+	capacity, err := units.RAMInBytes(size)
+	if err != nil {
+		return fmt.Errorf("failed to convert human readable size: %v into int64, err: %v", cr.Spec.Capacity, err)
+	}
+
+	stsObj, err = sts.NewBuilder().
+		WithName(cr.Name + "-jiva-rep").
+		WithLabelsNew(defaultReplicaLabels(cr.Spec.PV)).
+		WithNamespace(cr.Namespace).
+		WithServiceName("jiva-replica-svc").
+		WithPodManagementPolicy(appsv1.ParallelPodManagement).
+		WithStrategyType(appsv1.RollingUpdateStatefulSetStrategyType).
+		WithReplicas(&replicaCount).
+		WithSelectorMatchLabels(defaultReplicaLabels(cr.Spec.PV)).
+		WithPodTemplateSpecBuilder(
+			pts.NewBuilder().
+				WithLabels(defaultReplicaLabels(cr.Spec.PV)).
+				WithAffinity(&corev1.Affinity{
+					PodAntiAffinity: &corev1.PodAntiAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+							{
+								LabelSelector: &metav1.LabelSelector{
+									MatchLabels: defaultReplicaLabels(cr.Spec.PV),
+								},
+								TopologyKey: "kubernetes.io/hostname",
+							},
+						},
+					},
+				}).
+				WithContainerBuilders(
+					container.NewBuilder().
+						WithName("jiva-replica").
+						WithImage(getImage("OPENEBS_IO_JIVA_REPLICA_IMAGE",
+							"jiva-replica")).
+						WithPortsNew([]corev1.ContainerPort{
+							{
+								ContainerPort: 9502,
+								Protocol:      "TCP",
+							},
+							{
+								ContainerPort: 9503,
+								Protocol:      "TCP",
+							},
+							{
+								ContainerPort: 9504,
+								Protocol:      "TCP",
+							},
+						}).
+						WithCommandNew([]string{
+							"launch",
+						}).
+						WithArgumentsNew([]string{
+							"replica",
+							"--frontendIP",
+							fmt.Sprintf(svcNameFormat, cr.Name, cr.Namespace),
+							"--size",
+							fmt.Sprint(capacity),
+							"openebs",
+						}).
+						WithImagePullPolicy(corev1.PullIfNotPresent).
+						WithPrivilegedSecurityContext(&prev).
+						WithResources(&cr.Spec.ReplicaResource).
+						WithVolumeMountsNew([]corev1.VolumeMount{
+							{
+								Name:      "openebs",
+								MountPath: "/openebs",
+							},
+						}),
+				),
+		).
+		WithPVC(
+			pvc.NewBuilder().
+				WithGenerateName("openebs-jiva-replica-pvc").
```

Comment:
@prateekpandey14 Can you please look at this change.

Reply:
replacing this with older name "openebs", it will create the pvc's with following 
`openebs-pvc-19d05599-05d3-11ea-a004-42010a80019b-jiva-rep-0`
`openebs-pvc-19d05599-05d3-11ea-a004-42010a80019b-jiva-rep-1`
i.e, `openebs-<podname>`